profiles/sink/Sink_4K_TvSettings.yaml

```diff
@@ -610,9 +610,14 @@ tvSettings:
     range_from: 0
     range_to: 1000
     DimmingLevel:
-      - nonboost
-      - boost
-      - burst
+      range:
+        - nonboost
+        - boost
+        - burst
+      index:
+        - 0x00
+        - 0x01
+        - 0x02
```

Comment:
updated the whole file now, built and verified.
All the above issues are resolved.
Above issue came because some of the variables I was using find and replace and adding the initialization part as I came to the APIs